python/ray/serve/tests/test_logging.py

```diff
@@ -156,9 +164,12 @@ def __call__(self, *args):
             assert replica_tag not in f.getvalue()
 
 
-def test_context_information_in_logging(serve_instance):
+@pytest.mark.parametrize("json_log_format", ["0", "1"])
```

Comment:
```suggestion
@pytest.mark.parametrize("json_log_format", [True, False])
```
leaking implementation detail of the feature flag here

Reply:
good point.